src/test/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = {
+  root: true,
+  extends: 'fatfisz',
```

Comment:
Why Rafał R here? :D

Reply:
Oh I see - this is custom esLint profile provided by Rafał - makes sense.

Reply:
it works very well, I tested it on other projects